hooks/useFeeEstimation.ts

```
@@ -25,7 +25,9 @@
 
     const fee = calculateFee(
       Math.round(gasEstimation * (modifier || 1.5)),
-      GasPrice.fromString(`${gasPrice}${chainName === 'manifesttestnet' ? 'umfx' : 'uosmo'}`)
+      GasPrice.fromString(
+        `${gasPrice}${chainName === 'manifesttestnet' || 'manifestdevnet' ? 'umfx' : 'uosmo'}`
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)